MANIFEST.in

```
@@ -0,0 +1,2 @@
+include README.md
+include LICENSE
```

Do you want more things included in the python distribution? Namely, q2 README.md,
```suggestion
include LICENSE
include woltka/q2/README.md
```
Also, there are a number of files included as testing data, do you want these packaged with `woltka`?

You can see [this stackoverflow thread](https://stackoverflow.com/questions/24727709/do-python-projects-need-a-manifest-in-and-what-should-be-in-it) for some info on MANIFEST.in,
namely, you can check what has been included in your distribution by running
`python setup.py sdist` and inspecting the tarball in `dist`.

---

Good point! I have commited your suggestion. I am not sure whether the test data should be bundled in the package. If we do this, these files will appear at somewhere like `/Users/qiz173/conda/envs/woltka/lib/python3.7/site-packages/woltka`, which is quite instraightforward for average users. It is perhaps more straightforward for them to manually download the test data from the GitHub repo. I will add that instruction to `README.md`.

---

I think that it is good to package your test data (if it is reasonably small) with your package so that you can automatically test if your package works, by running the unittest suite.
That being said, if you decide to package it, it may also be good to add instructions to manually download the test data, as you mentioned, so they can try the tutorial with your test data, in a directory that is more straightforward to them and won't mess with data in their stie-packages.

---

Added more instructions in commit [f7f2b34](https://github.com/qiyunzhu/woltka/pull/1/commits/f7f2b34e295140b78406fdbe366412afde0c1d10).